scripts/obtain/dotnet-install.ps1

```diff
@@ -253,7 +253,7 @@ function Get-Download-Link([string]$AzureFeed, [string]$Channel, [string]$Specif
         $PayloadURL = "$AzureFeed/Runtime/$SpecificVersion/dotnet-runtime-$SpecificVersion-win-$CLIArchitecture.zip"
     }
     else {
-        $PayloadURL = "$AzureFeed/Sdk/$SpecificVersion/dotnet-dev-$SpecificVersion-win-$CLIArchitecture.zip"
+        $PayloadURL = "$AzureFeed/Sdk/$SpecificVersion/dotnet-sdk-$SpecificVersion-win-$CLIArchitecture.zip"
```

Comment:
The POR is to patch the 41 builds published at blob storage to work with the newer version of the scripts [we are referring to the version of the script in this PR]. Patching defined as adding a copy of the 'dotnet-dev' packages as 'dotnet-sdk' so that the newer version of the scripts will be completely backwards compatible. The range of builds are: 2.0.0-preview2-006115 thru 2.0.0-preview2-006353

**To be clear, any download after 2.0.0-preview2-006353 cannot be downloaded with the older version of the scripts.**